cli/lib/commands/init.js

```diff
@@ -0,0 +1,210 @@
+'use strict';
+
+module.exports = create;
+
+const readline = require('readline');
+const {
+  createPackageJson,
+  createToml,
+  fileExists,
+  isValidSemver,
+  writeFile
+} = require('../utils');
+
+const REJECTION_MSGS = {
+  REQUIRED: 'This is required',
+  validateName: "Sorry, that's not a valid name",
+  validateVersion: "Sorry, that's not a valid semver version"
+};
+
+const VALID_YES_NO = {
```

Comment:
I guess `1` is `true` and `0` is `false` like in good old `C` times? Well booleans are completely fine :+1: